paddle/fluid/framework/ir/graph.h

```diff
@@ -71,6 +72,7 @@ namespace ir {
 class Graph {
  public:
   explicit Graph(const ProgramDesc &program);
+  Graph(const Graph &o);
```

Comment:
can we use a `Clone` method and ban this method.
In the `Clone` comment, say that it only clones graph structure, not the attributes.

Reply:
OK. Thx.